nltk/tag/stanford.py

```diff
@@ -204,6 +205,79 @@ def parse_output(self, text, sentences):
 
         raise NotImplementedError
 
+class CoreNLPTagger(CoreNLPParser, TaggerI):
+    def __init__(self, url='http://localhost:9000', encoding='utf8'):
+        """
+        This is a abstract, duck-type of the CoreNLP that returns the
```

Comment:
I would not mention duck-typing (it's a bit confusing to me). Something like "An abstract interface to POS/NER taggers of CoreNLP" looks better to me.